sunpy/net/dataretriever/client.py

```diff
@@ -290,7 +279,8 @@ def search(self, *args, **kwargs):
                 return QueryResponse.create(self.map_, urls, times)
         return QueryResponse.create(self.map_, urls)
 
-    def fetch(self, qres, path=None, error_callback=None, **kwargs):
+    def fetch(self, qres, path=None, overwrite=False,
```

Comment:
Now that it defaults to not overwrite - can the progress bar not appear if file already exists and it should maybe print 'file exists /path' ?